client/my-sites/sharing/buttons/options.jsx

```diff
@@ -22,6 +22,10 @@ import { isJetpackSite, isJetpackMinimumVersion } from 'state/sites/selectors';
 import QueryPostTypes from 'components/data/query-post-types';
 import { recordGoogleEvent } from 'state/analytics/actions';
 
+{
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
this doesn't need to be in `{ }`, it can just be a normal block comment

Reply:
Thanks @blowery! Updated in: https://github.com/Automattic/wp-calypso/commit/b17178500992c6ac65320ba36e927c81cb53647f